project-base/storefront/components/Pages/Contact/ContactContent.tsx

```diff
@@ -62,72 +61,60 @@ export const ContactContent: FC = () => {
-                            textInputProps={{
-                                label: formMeta.fields.name.label,
-                                required: true,
-                                type: 'text',
-                                autoComplete: 'name',
-                            }}
-                        />
-                        <TextInputControlled
-                            control={formProviderMethods.control}
-                            formName={formMeta.formName}
-                            name={formMeta.fields.email.name}
-                            render={(textInput) => (
-                                <FormColumn className="lg:w-[calc(65%+0.75rem)]">
-                                    <FormLine bottomGap className="w-full flex-none lg:w-1/2">
-                                        {textInput}
-                                    </FormLine>
-                                </FormColumn>
-                            )}
-                            textInputProps={{
-                                label: formMeta.fields.email.label,
-                                required: true,
-                                type: 'email',
-                                autoComplete: 'email',
-                            }}
-                        />
-                        <TextareaControlled
-                            control={formProviderMethods.control}
-                            formName={formMeta.formName}
-                            name={formMeta.fields.message.name}
-                            render={(textarea) => (
-                                <FormColumn className="lg:w-[calc(65%+0.75rem)]">
-                                    <FormLine bottomGap className="w-full">
-                                        {textarea}
-                                    </FormLine>
-                                </FormColumn>
-                            )}
-                            textareaProps={{
-                                label: formMeta.fields.message.label,
-                                required: true,
-                                rows: 4,
-                            }}
-                        />
-                        <CheckboxControlled
-                            control={formProviderMethods.control}
-                            formName={formMeta.formName}
-                            name={formMeta.fields.privacyPolicy.name}
-                            render={(checkbox) => <ChoiceFormLine>{checkbox}</ChoiceFormLine>}
-                            checkboxProps={{
-                                label: formMeta.fields.privacyPolicy.label,
-                                required: true,
-                            }}
-                        />
-                        <SubmitButton isWithDisabledLook={!formProviderMethods.formState.isValid}>
-                            {t('Send message')}
-                        </SubmitButton>
+                        <FormWrapper>
```

Comment:
Why is FormWrapper not wrapping the form?

Reply:
Good point, first I'll explain why we don't put these styles to the Form component:
We don't always need to wrap the entire form with this component, for example ContactInformationWrapper

So it's just about naming, do you have a better name for this component in your mind?

Reply:
I mean, you could name it `FormContentWrapper` to show that it wraps the content, but I do not mind the current name either. Up to you. You can resolve as you wish.